app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command) == 3:
```

Comment:
Not correct check because "hello" will return true in this case
Command must contain THREE ARGUMENTS